operator/pkg/controllers/master/kubeapiserver.go

```diff
@@ -198,6 +199,19 @@ func apiServerPodSpecFor(controlPlane *v1alpha1.ControlPlane) v1.PodSpec {
 					MountPath: "/var/aws-iam-authenticator/kubeconfig/",
 					ReadOnly:  true,
 				}},
+				LivenessProbe: &v1.Probe{
+					Handler: v1.Handler{
+						HTTPGet: &v1.HTTPGetAction{
+							Host:   "127.0.0.1",
+							Scheme: v1.URISchemeHTTPS,
+							Path:   "/healthz",
```

Comment:
likewise, readiness vs liveness: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
Will move away from healthz endpoint for apiserver and use readyz for readiness and livez for liveness.